clusterloader2/testing/density/config.yaml

```diff
@@ -25,22 +33,23 @@ steps:
 - phases:
   - namespaceRange:
       min: 1
-      max: 3
+      max: {{$namespaces}}
     replicasPerNamespace: 1
     tuningSet: Uniform100qps
     objectBundle:
     - basename: saturation-rc
       objectTemplatePath: rc.yaml
       templateFillMap:
-        Replicas: 30
+        Replicas: {{$replicas}}
         PodKind: saturationPod
 - measurements:
   - Identifier: WaitForSaturationRunning
     Method: WaitForRunningPods
     Params:
-      desiredPodCount: 90
+      desiredPodCount: {{MultiplyInt $namespaces $replicas}}
```

Comment:
Isn't this just `$totalpods`?

Reply:
Or you're trying to account for rounding off?

Reply:
Then instead of doing this multiplication at places, could you instead just compute a `$totalPodsRounded` at the top and use it?

Reply:
Done.